UI/SidePanel.cpp

```diff
@@ -2535,6 +2542,15 @@ void SidePanel::PlanetPanelContainer::SetPlanets(const std::vector<int>& planet_
         GG::Connect(m_planet_panels.back()->RightClickedSignal,         PlanetRightClickedSignal);
         GG::Connect(m_planet_panels.back()->BuildingRightClickedSignal, BuildingRightClickedSignal);
         GG::Connect(m_planet_panels.back()->ResizedSignal,              &SidePanel::PlanetPanelContainer::RequirePreRender,       this);
+        m_planet_panels.back()->OrderButtonChangedSignal.connect(
+            [this](int planet_id) {
```

Comment:
Infinite loop?  We need to call `RequirePreRender` to update the layout.
`SidePanel::PreRender` -> `SidePanel::UpdateImpl` -> `PlanetPanelContainer::RefreshAllPlanetPanels`